app/javascript/components/Background.js

```diff
@@ -0,0 +1,91 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import Loader from 'react-loader-spinner'
+
+import BackgroundList from './BackgroundList'
+import UploadButton from './UploadButton'
+
+import "react-loader-spinner/dist/loader/css/react-spinner-loader.css"
+import '../../assets/stylesheets/account/backgrounds.scss'
+
+export default class Background extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      backgrounds: [],
+      imageSpinner: true
+    };
+
+    this.fetchImages = this.fetchImages.bind(this);
+    this.onDeleteClick = this.onDeleteClick.bind(this);
+    this.uploadFile = this.uploadFile.bind(this);
+  }
+
+  componentDidMount() {
+    this.fetchImages();
+  }
+
+  fetchImages() {
+    fetch('/account/backgrounds', {
+      headers: {
+        'Content-Type': 'application/json',
+        'Accept': 'application/json'
```

Comment:
fyi: in rails app you simply can add `.json` to your url as `fetch('/account/backgrounds.json')` and no need for headers